scripts/sysman/memoryTimestampValidBits.yml

```diff
@@ -32,10 +32,11 @@ class: $sMemory
 name: $s_mem_timestamp_bits_exp_t
 details:
     - "Number of counter bits"
+    - "1.8": "[DEPRECATED] No longer supported."
```

Comment:
agree with @joshuaranjan that we could remove it since it's experimental. 
@wdamon-intel  what do you think? given we state in spec for experimental extensions that
`may appear and disappear from release to release`

Reply:
this is experimental, we can just remove it. Or if the new extension is fixing this one, why not just do all on top of this one?

Reply:
@jandres742, I will remove the old one and retain new one(what we return here is bandwidth counter valid bits and not timestamp valid bits).old one need filename change too.